resources/migrations/001_update_migrations.yaml

```diff
@@ -9366,7 +9366,7 @@ databaseChangeLog:
                     nullable: false
 
   - changeSet:
-      id: v51.2024-09-05T08:00:03
+      id: v51.2024-09-05T08:00:02
```

Comment:
You'll need to cherry-pick this to the previous PR to avoid breaking Stats I think.